src/otx/algo/instance_segmentation/maskrcnn.py

```diff
@@ -350,3 +392,83 @@ def _optimization_config(self) -> dict[str, Any]:
             return {"model_type": "transformer"}
 
         return {}
+
+
+class RotatedMaskRCNNModel(MaskRCNN):
```

Comment:
For the next refactor; It would be nice to split these files as well

Reply:
RotatedMaskRCNNModel is actually MaskRCNN, but with an additional postprocessing on the predict stage. We discussed that with Eugene and decided to put this model alongside with MaskRCNN.